src/Umbraco.TestData/LoadTestController.cs

```diff
@@ -44,7 +44,7 @@ public class LoadTestController : Controller
 <body>
   <div class=""head"">
   <h1><a href=""/LoadTest"">LoadTest</a></h1>
-  <div class=""ver"">" + System.Configuration.ConfigurationManager.AppSettings["umbracoConfigurationStatus"] + @"</div>
+  <div class=""ver"">@_umbracoVersion.SemanticVersion</div>
```

Comment:
Not sure if it is required, but guess you would like to use the `ToSemanticStringWithoutBuild` extension method.

Otherwise, it returns the build info you added in another PR, to make github source available IIRC.